db/migrate/20220317025428_update_bootstraped_subdomain_hostname.rb

```diff
@@ -0,0 +1,8 @@
+class UpdateBootstrapedSubdomainHostname < ActiveRecord::Migration[6.1]
+  def change
+    Apartment::Tenant.switch('public') do
```

Comment:
@Pralish this should be in `up` and `down` blocks in case people have interesting configurations that might break if `www` is removed ;)